src/Heading/index.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import * as React from "react";
+import * as tokens from "orbit-design-token";
+
+const weightHeading = {
+  medium: tokens.fontWeightMedium,
+  bold: tokens.fontWeightBold,
+};
+
+const sizeHeading = {
+  display: tokens.fontSizeHeadingDisplay,
+  large: tokens.fontSizeHeadingLarge,
+  medium: tokens.fontSizeHeadingMedium,
+  small: tokens.fontSizeHeadingSmall,
+};
+
+type Props = {
+  element: "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "div",
```

Comment:
can you pls use only `h1` `h2` `h3` for now ? talked in person with @darkwindcz about

Reply:
Yes, no problem.